### PR TITLE
Media upgrade bugfixing

### PR DESCRIPTION
**Pull request by gugalo**
**Jira issue:**
https://glia.atlassian.net/browse/MOB-1978

**Additional info:**
- Fixed wrong media upgrade messages in Call Visualizer
- Added check for Call Visualizer not to request camera permission for a one-way video stream
- Fixed system mic/camera/screen-sharing permissions not being requested on API 28 and lower

### CODE REVIEW

**Review by Pelkar — APPROVED** [review]
LGTM 👍

**Comment by gugalo** [comment]
One of my changes broke the tests. Please check this commit: https://github.com/salemove/android-sdk-widgets/compare/d32b3d4dca4c7415b0a8039b6ab6f00398d4ab7c..6bac839a85eb3fab13c950c43f7b943cf00edffe

**Review by DavDo — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/CallVisualizerController.kt [hunk 1]
```diff
@@ -43,12 +43,12 @@ internal class CallVisualizerController(
 
     override fun onOneWayMediaUpgradeRequest(mediaUpgradeOffer: MediaUpgradeOffer, operatorName: String) {
         val formattedOperatorName = Utils.formatOperatorName(operatorName)
-        dialogController.showUpgradeVideoDialog2Way(mediaUpgradeOffer, formattedOperatorName)
+        dialogController.showUpgradeVideoDialog1Way(mediaUpgradeOffer, formattedOperatorName)
```
Comment (AndriiHorishniiMOC) [inline widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/CallVisualizerController.kt:46]: Wow! So it was working vice versa until now?

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/CallVisualizerController.kt (hunk 1 above)
Comment (andrews-moc) [reply widgetssdk/src/main/java/com/glia/widgets/callvisualizer/controller/CallVisualizerController.kt:46]: Yes, that's good that @gugalo has noticed that. 👍 Thank you.

**Review by andrews-moc — APPROVED** [review]

**Comment by gugalo** [comment]
!squash

**Comment by gugalo** [comment]
!merge

**Pull request merged**